qipr_approver/approver/views/index.py

```diff
@@ -5,12 +5,12 @@
 from django.contrib.auth.models import User
 from approver.models import Project
 
-import json
-import approver.utils as utils
 import approver.constants as constants
+import approver.utils as utils
+from approver.views.dashboard import dashboard
 
 def index(request):
-    context = {
-        'content': 'approver/index.html',
-    }
-    return utils.layout_render(request, context)
+    if(utils.get_current_user_gatorlink(request.session)):
+        return dashboard(request)
```

Comment:
Anything that lets you send http requests. You could jump into a python shell, import requests and go to town. Inspect what we are sending with the chrome network tab and try to duplicate that